test/conformance/enqueue/urEnqueueKernelLaunch.cpp

```diff
@@ -295,6 +295,68 @@ TEST_P(urEnqueueKernelLaunchTestWithParam, Success) {
     ValidateBuffer(buffer, buffer_size, val);
 }
 
+struct urEnqueueKernelLaunchWithUSM : uur::urKernelExecutionTest {
```

Comment:
I'd like to see tests that allocate stuff in one thread, and use it in another, and then deallocate in yet another. But we can leave that as TODO.

Reply:
I'll create a task for that.